olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -1394,5 +1394,5 @@ spec:
       image: "registry.ci.openshift.org/openshift/knative-eventing-kafka-broker-post-install:knative-v1.11"
     - name: "KAFKA_IMAGE_knative-kafka-storage-version-migrator__migrate"
       image: "registry.ci.openshift.org/openshift/knative-eventing-migrate:knative-v1.11"
-  replaces: serverless-operator.v1.32.0
-  version: 1.32.1
+  replaces: serverless-operator.v1.32.1
```

Comment:
Not sure if the "replaces" thing should stay at 1.32.0 or should be bumped. We did a different thing with 1.30.2: https://github.com/openshift-knative/serverless-operator/commit/db0039bef8b2ea4a55cce305e66366c1f5d02765#diff-55554988a6ad1a56cfb485f9de15784449897484786cdbfcd8c3ffbb97be92e1R12